src/multilspy/language_servers/kotlin_language_server/runtime_dependencies.json

```diff
@@ -0,0 +1,53 @@
+{
+    "_description": "Used to download the runtime dependencies for Kotlin Language Server from https://github.com/fwcd/kotlin-language-server",
+    "runtimeDependency": {
+        "id": "KotlinLsp",
+        "description": "Kotlin Language Server",
+        "url": "https://github.com/fwcd/kotlin-language-server/releases/download/1.3.13/server.zip",
+        "archiveType": "zip"
+    },
+    "jdk": {
+        "win-x64": {
+            "url": "https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21.0.6%2B7/OpenJDK21U-jdk_x64_windows_hotspot_21.0.6_7.zip",
```

Comment:
Can you say if any of these can be replaced by the repos and urls at https://github.com/microsoft/multilspy/blob/127de628c87b26a5c3077a44e2b6ba80e23e5651/src/multilspy/language_servers/eclipse_jdtls/runtime_dependencies.json#L12